packages/clerk-js/src/ui/components/Checkout/parts.tsx

```diff
@@ -82,7 +82,11 @@ export const InvalidPlanScreen = () => {
           <Alert
             variant='info'
             colorScheme='info'
-            title={localizationKeys('commerce.cannotSubscribeMonthly')}
+            title={
+              planFromError.annual_monthly_amount_formatted !== '0'
```

Comment:
We probably want to expose a numeric amount to check against here.

Reply:
Let's use number here, what happens if string comes back as `0.00` ?
```suggestion
              planFromError.annual_monthly_amount !== 0
```

If the FAPI does not return numbers, we should update it.